superset/cli.py

```diff
@@ -133,9 +133,15 @@ def load_examples(load_test_data):
     print("Loading [Misc Charts] dashboard")
     data.load_misc_dashboard()
 
+    print("Loading DECK.gl demo")
```

Comment:
merge resolution mismatch?

Reply:
still WiP, I'll ping when ready for review...